UI/src/view/GroupedLinearAssetLayer.js

```diff
@@ -0,0 +1,92 @@
+(function (root) {
+  root.GroupedLinearAssetLayer = function (params, map) {
+
+    var isActive = params.massLimitation;
+    var style = params.style;
+    var collection = params.collection;
+    var typeId = params.typeId;
+    var isComplementaryChecked = false;
+
+    var uiState = { zoomLevel: 9 };
+
+    var vectorSource = new ol.source.Vector();
+    var vectorLayer = new ol.layer.Vector({
+      source : vectorSource,
+      style : function(feature) {
+        return style.browsingStyleProviderReadOnly.getStyle(feature, {zoomLevel: uiState.zoomLevel});
+      }
+    });
+
+    vectorLayer.set('name', 'GroupedLinearAssetLayer');
+    vectorLayer.setOpacity(1);
+    vectorLayer.setVisible(false);
+    map.addLayer(vectorLayer);
+
+    var adjustStylesByZoomLevel = function(zoom) {
+      uiState.zoomLevel = zoom;
+    };
+
+    var showLayer = function(){
+      if(isActive)
+        vectorLayer.setVisible(true);
+    };
+
+    var hideLayer = function(){
+      vectorLayer.setVisible(false);
+    };
+
+    var showWithComplementary = function () {
+      isComplementaryChecked = true;
+    };
+
+    var hideComplementary = function(){
+      isComplementaryChecked = false;
+    };
+
+    var removeLayerFeatures = function(){
+      vectorLayer.getSource().clear();
+    };
+
+    var redrawLinearAssets = function(linearAssetChains) {
+      vectorSource.clear();
+      var linearAssets = _.flatten(linearAssetChains);
+      drawLinearAssets(linearAssets);
+    };
+
+    var offsetBySideCode = function (linearAsset) {
+      return GeometryUtils.offsetBySideCode(applicationModel.zoom.level, linearAsset);
+    };
+
+    var drawLinearAssets = function(linearAssets) {
+      var asset = _.filter(linearAssets, function(asset) { return !_.some(asset.values, function(type) { return type.typeId == typeId; }); });
+      vectorSource.addFeatures(new MassLimitationsLabel().renderFeaturesByLinearAssets(_.map(_.cloneDeep(linearAssets), offsetBySideCode), uiState.zoomLevel));
```

Comment:
We can substitute this initialization of MassLimitationLabel for the params.assetLabel and add the MassLimitationLabel to the asset configuration file